src/shared/common/forms/accordion.jsx

```diff
@@ -12,6 +12,7 @@ function Section({ data, errors, index, items, title, updateState, page }) {
     const sectionHasValidationError = errors && Array.isArray(items) && items.some(itemHasValidationError(errors));
     const [isVisible, setVisible] = useState(sectionHasValidationError);
     const [isFocussed, setFocussed] = useState(false);
+    const [toggleText, setToggleText] = useState('Show');
```

Comment:
You're switching between boolean and string here, which could cause confusion.

Reply:
Where is it setting as boolean sorry I can't spot it? Here it's initialising toggleText as string 'Show' and further down toggleText is set to string 'Hide.

Reply:
ah, I am looking at `useState(false)` and misreading line 14.